tests/frontend/tests/test_pools.py

```diff
@@ -0,0 +1,126 @@
+import pytest
+from jumpscale.loader import j
+from tests.frontend.pages.pools.pools_solution import PoolsSolution
+from tests.frontend.pages.wallets.wallets import Wallets
+from tests.frontend.pages.pools.pools import Pools
+from tests.frontend.tests.base_tests import BaseTest
+
+
+@pytest.mark.integration
+class PoolsTests(BaseTest):
+    def test01_create_pool(self):
+        """Test case for creating a pool.
+
+        **Test Scenario**
+        - Create a wallet.
+        - Create a pool.
+        - Check that the pool has been created in the pools card.
+        - Delete the a wallet.
+        """
+        self.info("Create a wallet")
+        wallet_name = self.random_name()
+        wallets = Wallets(self.driver)
+        wallets.load()
+        wallets.add_funded(wallet_name)
+
+        self.info("Create a pool")
+        pool_name = self.random_name().lower()
+        pools = PoolsSolution(self.driver)
+        pools.load()
+        pools.create(name=pool_name, wallet_name=wallet_name)
```

Comment:
Please create the pool with random values for cus and sus to make sure that these values reflect correctly

Reply:
Then we can check also the pool has been created with the right amount of cus and sus